### PR TITLE
generate null funcs

### PR DESCRIPTION
**Pull request by xmclark**
This PR automates adding the `nullfunc` imports. A new macro is added to help working with namespaces. All functions in the env namespace that start with `nullFunc_` will be added to the import object. This is easy because *all* null funcs have the same expected signature.

### CODE REVIEW

**Comment by xmclark** [comment]
bors try

File: lib/runtime-core/src/macros.rs [hunk 1]
```diff
@@ -75,3 +75,15 @@ macro_rules! __imports_internal {
         $ns
     };
 }
+
+#[macro_export]
+#[doc(hidden)]
+macro_rules! ecapseman {
```
Comment (bjfish) [inline lib/runtime-core/src/macros.rs]: What does `ecapseman` mean?

File: lib/emscripten/src/lib.rs [hunk 2]
```diff
@@ -770,7 +730,15 @@ pub fn generate_emscripten_env(globals: &mut EmscriptenGlobals) -> ImportObject
         "asm2wasm" => {
             "f64-rem" => func!(crate::math::f64_rem),
         },
-    }
+    };
+
+    import_object
+}
+
+pub fn nullfunc_x(ctx: &mut Ctx, _x: u32) {
```
Comment (syrusakbary) [inline lib/emscripten/src/lib.rs]: Should be just `nullfunc`? (without an x)

```
pub fn nullfunc_x(ctx: &mut Ctx) {
...
}
```

Comment (xmclark) [reply lib/emscripten/src/lib.rs]: sure, sound good

**Pull request merged**